src/atomate2/classical_md/__init__.py

```diff
@@ -0,0 +1,70 @@
+"""Module for classical md workflows."""
+
+from openff.interchange import Interchange
+from openff.toolkit.topology import Topology
+from openff.toolkit.topology.molecule import Molecule
+from openff.units import Quantity
+
+
+def openff_mol_as_monty_dict(self: Molecule) -> dict:
+    """Convert a Molecule to a monty dictionary."""
+    mol_dict = self.to_dict()
+    mol_dict["@module"] = "openff.toolkit.topology"
+    mol_dict["@class"] = "Molecule"
+    return mol_dict
+
+
+Molecule.as_dict = openff_mol_as_monty_dict
```

Comment:
I'm not sure how I feel about this. Seems like it could break easily? The other options are: 
1. Add as_dict methods to openff classes for monty support.
2. Add support for openff to monty (doesn't seem likely given we tried to do something similar for ase and it was turned down).
3. Some kind of plugin system for monty where we can define some common serialisation routes for materials science objects.

For now we can leave this as is but I'd like to explore option 3 further (cc'ing @Andrew-S-Rosen @janosh @mkhorton)

Reply:
Agreed that it's not a perfect solution.

I think I'd be turned down for 1 & 2 both. I'd be happy to support option 3 in the future.